lib/learn_open/lessons/base_lesson.rb

```diff
@@ -45,16 +45,6 @@ def readme?
         !lab?
       end
 
-      def warn_if_necessary
```

Comment:
@StevenNunez cool, i like that it's the environment's responsibility to know whether or not to throw this warning, not the lesson's